.circleci/config.yml

```diff
@@ -462,27 +453,23 @@ jobs:
                 name: Deploy snyk-monitor resource
             - run:
                 command: |
+                    source $BASH_ENV
                     set -eo pipefail
 
                     REPLACES_VERSION=${OPERATOR_VERSION}
 
                     OPERATOR_DIR=$(python scripts/operator/package_operator_bundle.py "${LATEST_TAG}" "${LATEST_TAG}" "${LATEST_TAG}" "${REPLACES_VERSION}")
-
-                    export QUAY_TOKEN=$(python ./scripts/operator/get_quay_token.py "${QUAY_USERNAME}" "${QUAY_PASSWORD}")
-
-                    QUAY_NAMESPACE=snyk-runtime
-                    PACKAGE_NAME=snyk-operator
-
-                    set +x
-                    operator-courier push "${OPERATOR_DIR}" "${QUAY_NAMESPACE}" "${PACKAGE_NAME}" "${LATEST_TAG}" "${QUAY_TOKEN}"
-                    set -x
-                name: Package Operator Bundle upgrade and push to Quay
+                    python scripts/operator/create_operator_bundle_and_index_and_push.py $OPERATOR_DIR $LATEST_TAG $DOCKERHUB_USER $DOCKERHUB_PASSWORD $REPLACES_VERSION
+                description: Now we are testing that any upgrades to the Operator are detected and applied in the cluster.
+                name: Package Operator Bundle upgrade and push to Docker Hub
             - run:
                 command: |
+                    source $BASH_ENV
```

Comment:
Why we need to source it everytime? Isn't it's sources by default on each command?

Reply:
No idea but it didn't work without it